portfolio/target/portfolio-1/projects.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Projects</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+    <h1>Projects</h1>
+        <p> While I’m always doing projects in class (all in C and organized on <a href = "https://github.com/nisom1">GitHub</a>), below are some projects I pursued outside of the classroom that influenced my passions and goals. They're organized similar to a typical resume, with the most recent projects at the top. </p>
+    <h2>Diversity and Inclusion Research with Purdue University’s College of Electrical and Computer Engineering (ECE) (Summer 2019)</h2>
+        <p> • Worked with Purdue's College of ECE’s Diversity and Inclusion Committee to identify initiatives and actions that are contributing to the differing levels of diversity and inclusion between engineering disciplines (why might one discipline be more inclusive than another). </p>
```

Comment:
For bullet points, you can use HTML's &lt;ul&gt; and &lt;li&gt; tags, which helps the browser better understand the structure of the webpage.